hokusai/staging.yml

```diff
@@ -79,9 +79,9 @@ spec:
         resources:
           requests:
             cpu: 100m
-            memory: 256Mi
-          limits:
             memory: 512Mi
+          limits:
+            memory: 768Mi
```

Comment:
Amount of memory available for the container. Want it to be > `512` to give overhead for non-node processes.